modules/client/src/types.ts

```diff
@@ -422,3 +422,15 @@ export type InitializeFromParams = {
   previousVersion: [number, number, number];
   initData?: Uint8Array;
 };
+
+export type DaoUpdateProposalValidity = {
+  isValid: boolean;
+  causes: DaoUpdateProposalValidityCause[];
+};
+
+export enum DaoUpdateProposalValidityCause {
```

Comment:
Those are invalidity cause, not validity causes. (Every item below starts with "INVALID_")